apps/src/code-studio/progressRedux.js

```diff
@@ -595,7 +596,7 @@ const levelWithStatus = (
     status: statusForLevel(level, levelResults),
     isCurrentLevel: currentLevelId === normalizedLevel.id,
     paired: levelPairing[level.activeId],
-    locked: levelProgress?.locked,
+    isLocked: levelProgress?.locked,
```

Comment:
Not sure if this is important, but isLocked can be undefined here.

Reply:
nice catch. i made that a required property, so i will add `|| false`